### PR TITLE
Refactor formatting

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

**Review by hexonaut — DISMISSED** [review]
LGTM

File: test/CrosschainTestBase.sol [hunk 1]
```diff
@@ -236,27 +244,63 @@ abstract contract CrosschainTestBase is Test  {
...
+            false
+        );
+        assertEq(
+            IL2BridgeExecutor(bridgeExecutor).getActionsSetById(1).executed,
+            false
+        );
+        assertEq(
+            IL2BridgeExecutor(bridgeExecutor).getActionsSetById(2).executed,
+            false
+        );
```
Comment (lucas-manuel) [inline test/CrosschainTestBase.sol:258]: These can be left as is imo, looks better

File: test/GnosisCrosschainTest.t.sol [hunk 2]
```diff
@@ -20,26 +20,24 @@ import { CrosschainPayload, CrosschainTestBase } from './CrosschainTestBase.sol'
...
         XChainForwarders.sendMessageGnosis(
             bridgeExecutor,
             encodeCrosschainExecutionMessage(),
             1_000_000
         );
     }
-
 }
 
 contract GnosisCrosschainTest is CrosschainTestBase {
```
Comment (lucas-manuel) [inline test/GnosisCrosschainTest.t.sol:35]: Leave new line

File: test/GnosisCrosschainTest.t.sol [hunk 3]
```diff
@@ -106,18 +107,11 @@ contract GnosisCrosschainTest is CrosschainTestBase {
...
         assertEq(
             GnosisBridgeExecutor(bridgeExecutor).controller(),
             newController
         );
-        assertEq(
-            GnosisBridgeExecutor(bridgeExecutor).chainId(),
-            newChainId
-        );
+        assertEq(GnosisBridgeExecutor(bridgeExecutor).chainId(), newChainId);
     }
```
Comment (lucas-manuel) [inline test/GnosisCrosschainTest.t.sol:117]: Leave new line

**Review by lucas-manuel — CHANGES_REQUESTED** [review]
Since we're doing it can we update all imports that are doing `{this}` to do `{ this }` as well?

File: test/CrosschainTestBase.sol (hunk 1 above)
Comment (barrutko) [reply test/CrosschainTestBase.sol:258]: Frankly, I prefer the multiline formatting for asserts, as it's more visually clear what two values are being compared.
If you have a super strong opinion here though, I can change it back to single line.

**Review by lucas-manuel — APPROVED** [review]

**Review by hexonaut — APPROVED** [review]
LGTM

**Pull request merged**